### ISSUE TITLE
When someone is trying to generate a new key, they receive an erroneous error.  This confuses new validators

### ISSUE THREAD

**Issue by OgreAbroad** [harmony-one/go-sdk#241]
When the following command is issued: (In accordance with docs.harmony.one)

./hmy keys generate-bls-keys --count 1 --shard 1 --passphrase

It returns an erroneous error:

Cannot connect to node http://localhost:9500, using Harmony mainnet endpoint https://api.s0.t.hmny.io/

If paying attention, it still allows you to create a passphrase, repeat to verify, and generate the key.  In the past week, 2 out of 4 new validators got tripped up by this step.

**Comment by sophoah** [comment]
@SebastianJ @rlan35 do you know what is the purpose of that error message ? I feel like it doesn't need to be specified ? also i never understood why there is need to link a BLS key to a network ? Creating the BLS key should be able to done offline.

**Comment by MendoCrypto** [comment]
Yeah, this stumbled me for sure. I destroyed my VPS and started over. But it's pretty much par the course when setting up a new node to destroy things and start over several times. It's always easier than trying to identify and fix some little thing you may have broken.

**Comment by LeoHChen** [comment]
Maybe we just need to improve the help message a bit more.
